### DISCUSSION TITLE
Hello everyone, please help to see the following error is what？

### DISCUSSION THREAD

**Discussion by Pythoneryeah** [coder/code-server#6336] [Q&A]
Error: Unexpected SIGPIPE
    at process.<anonymous> (/usr/lib/code-server/vendor/modules/code-oss-dev/out/bootstrap.js:1:370)
    at process.emit (events.js:400:28)

**Comment by code-asher** [comment]
It means that the process tried to write to a pipe without a 
process connected to the other end.  It could be a bug but I am 
not entirely sure what would cause this.  Is there anything 
unusual you were doing at the time or do you have reproduction 
steps?  And, are you seeing this error in the console or in VS 
Code itself?
